packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -297,16 +334,49 @@ class PageView extends StatefulWidget {
     assert(childrenDelegate != null);
   }
 
+  /// The axis along which the page view scrolls.
+  ///
+  /// Defaults to [Axis.horizontal].
   final Axis scrollDirection;
 
+  /// Whether the page view scrolls in the reading direction.
+  ///
+  /// For example, if the reading direction is left-to-right and
+  /// [scrollDirection] is [Axis.horizontal], then the page view scrolls from
+  /// left to right when [reverse] is false and from right to left when
+  /// [reverse] is true.
+  ///
+  /// Similarly, if [scrollDirection] is [Axis.vertical], then the page view
+  /// scrolls from top to bottom when [reverse] is false and from bottom to top
+  /// when [reverse] is true.
+  ///
+  /// Defaults to false.
   final bool reverse;
 
+  /// An object that can be used to control the position to which this page
+  /// view is scrolled.
   final PageController controller;
 
+  /// How the page view should respond to user input.
+  ///
+  /// For example, determines how the page view continues to animate after the
+  /// user stops dragging the page view.
+  ///
+  /// The physics are modified to snap to page boundaries using
+  /// [PageScrollPhysics] prior to being used.
+  ///
+  /// Defaults to matching platform conventions.
   final ScrollPhysics physics;
 
+  /// Called whenever the page visible in the viewport changes.
```

Comment:
Since more than one page can be visible per PageController.viewFraction maybe refer to the page in the center of the viewport